src/libsync/propagateuploadv1.cpp

```diff
@@ -209,7 +204,7 @@ void PropagateUploadFileV1::slotPutFinished()
         }
         QByteArray replyContent;
         QString errorString = job->errorStringParsingBody(&replyContent);
-        qDebug() << replyContent; // display the XML error in the debug
+        qCDebug(lcPropagateUpload) << replyContent; // display the XML error in the debug
```

Comment:
`PropagateItemJob::done` will normally print errorString, I changed it to a warning when it's an error status.